lib/src/component/dom_components.dart

```diff
@@ -45,7 +45,7 @@ class DomProps extends component_base.UiProps with DomPropsMixin, ReactPropsMixi
   final Map props;
 }
 
-class SvgProps extends component_base.UiProps with DomPropsMixin, ReactPropsMixin, SvgPropsMixin implements DomProps {
+class SvgProps extends component_base.UiProps with DomPropsMixin, SvgPropsMixin implements DomProps {
```

Comment:
This is part of "_Fix mixing in the same props class twice_"; `ReactPropsMixin` is already mixed in by `component_base.UiProps`